lib/api/core/plugins/pluginsManager.js

```diff
@@ -1063,6 +1065,29 @@ class PluginsManager {
             : {}
         };
 
+      // load customs errors configuration file
+      if (!plugin.manifest.raw.errors) {
+        this.kuzzle.log.warn(`[${plugin.manifest.name}] This plugin doesn't have customs errors in the manifest file.`);
```

Comment:
Why do we need to warn that the plugin has no custom errors? They might just not be necessary, no?

Reply:
That is why it's just a warning

Reply:
I agree with @xbill82 here

Reply:
I removed it